Makefile

```diff
@@ -14,9 +14,30 @@ compile:
 dryRelease:
 	./gradlew bintrayUpload -PbintrayUser=dryUser -PbintrayKey=dryKey
 
-# deploy the current build to bintray, jcenter and maven central
+# deploy the current build to bintray, jcenter and maven central, upload debug symbols to symbol collector
 doRelease:
-	./gradlew bintrayUpload -PbintrayUser="$(BINTRAY_USERNAME)" -PbintrayKey="$(BINTRAY_KEY)" -PmavenCentralUser="$(MAVEN_USER)" -PmavenCentralPassword="$(MAVEN_PASS)" -PmavenCentralSync=true -PdryRun=false --info
+	# bintray/jcenter upload
+	./gradlew bintrayUpload -PbintrayUser="$(BINTRAY_USERNAME)" -PbintrayKey="$(BINTRAY_KEY)" -PdryRun=false --info
+
+	# upload symbols
+	cd sentry-android-ndk
+	# be sure SymbolCollector.Console is in your $PATH - https://github.com/getsentry/symbol-collector/releases
+	SymbolCollector.Console --upload directory --path ./ --batch-type android --bundle-id sentry-android-ndk-$VERSION --server-endpoint https://symbol-collector.services.sentry.io
```

Comment:
Maybe throw this as the last step so if we don't have this on the path it's 1 thing only to fix and rerun instead of all the other jfrog stuff after

Reply:
I thought about that but tbh I'd prefer that we upload artifacts and symbols which is almost never-failing and only later the maven central pain.
we could even split it up to a different make command for example.
`make mc sync` or whatever, so we don't bound a task that is always successful with another one that is always broken

Reply:
although my comment above is not really about your comment which is not having in the `path`, maybe the script could test if `SymbolCollector.Console` and `jfrog` are valid cms even before starting the whole thing